Assets/Scripts/CameraShake.cs

```diff
@@ -56,7 +56,7 @@ public class CameraShake : MonoBehaviour{
     }
 
     // Other varibles like frequency could be add to this StartShake function later. 
-    [Button]
+    //[Button]
```

Comment:
Was there a problem with Easy Buttons? I see your messages in help, but I wasn't in a situation to help you at that time. I can work with you to see what the problem is.

Reply:
I couldn't find it in package manager, but ever since I upgraded to 2020.1.13f I can't access package manager. It shows me this.
![Package_Manager_Problem](https://user-images.githubusercontent.com/61594356/111020351-0c448d00-837a-11eb-863c-856a3c2e8ed3.PNG)